templates/fleetdb-service.yaml

```diff
@@ -0,0 +1,13 @@
+kind: Service
+apiVersion: v1
+metadata:
+  labels:
+    k8s-service: fleetdb
+  name: fleetdb
+spec:
+  ports:
+    - name: "8000"
```

Comment:
I know serverservice had its name be "8000" but could we change this to "fleetdb-service' or something to be more inline with other services like conditionorc-api-service.yaml